packages/phone-number-privacy/combiner/.env

```diff
@@ -1,7 +1,7 @@
 # Options: json, human (default), stackdriver
 LOG_FORMAT=stackdriver
 # Options: fatal, error, warn, info (default), debug, trace
-LOG_LEVEL=info
+LOG_LEVEL=trace
```

Comment:
Was this change just made for debugging?

Reply:
Yea. I'll change it back once I'm sure all the logging looks good